wemake_python_styleguide/visitors/ast/naming.py

```diff
@@ -109,6 +112,17 @@ def check_attribute_name(self, node: ast.ClassDef) -> None:
                     naming.UpperCaseAttributeViolation(target, text=target.id),
                 )
 
+    @property
+    def variable_names_blacklist(self) -> Set[str]:
+        if self._variable_names_blacklist is None:
+            self._variable_names_blacklist = {
+                *VARIABLE_NAMES_BLACKLIST,
+                *self._options.forbidden_domain_names,
+            }
+            for name in self._options.allowed_domain_names:
```

Comment:
Can we use just `set` arithmetic here?

Reply:
Shouldn't be an error if `allowed` and `forbidden` names intersects?

Reply:
At least, if option variables intersect.

Reply:
Yes, I agree. But! Only during validation. Not in runtime

Reply:
is [`_ValidatedOptions.validate_options()`](https://github.com/wemake-services/wemake-python-styleguide/blob/master/wemake_python_styleguide/options/validation.py#L72) proper place to add above mentioned validation logic?

Reply:
Yeap!